### ISSUE TITLE
You are using "MANAGE_EXTERNAL_STORAGE" permission in your app.

### ISSUE THREAD

**Issue by NomanBaigA** [Mastersam07/wa_status_saver#23]
It means google is not going to accept that app in the play store

**Comment by bhavikgajipara11** [comment]
> 

i am having issue publishing app on play store. do you have any solution?

**Comment by erKaranjeet** [comment]
Right, You are using MANAGE_EXTERNAL_PERMISSION and google will never approve this. You have to try without this permission. I'm facing the same problem and finding the solution of it.

**Comment by bhavikgajipara11** [comment]
only one way it is possible it it to use native code in flutter app. i tried kotlin code but when i tried to sync gradle it was syncing for 3 hrs and failed.

**Comment by 1MinuteAunty** [comment]
Hi, I have encountered a similar issue and managed to find a solution. In my project from [statussaver](https://github.com/krupal4/statussaver), I addressed this by [using Storage Access Framework instead of MANAGE_EXTERNAL_STORAGE]. 

I hope this helps!

**Comment by 1MinuteAunty** [comment]
@bhavikgajipara11  @erKaranjeet @NomanBaigA @Mastersam07
